scripts/issue/1122/create_gtc_position.py

```diff
@@ -0,0 +1,41 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+LOWER_RANGE_MULTIPLIER = 0.9
+UPPER_RANGE_MULTIPLIER = 2
+GTC_SELL_PCT = 0.16
```

Comment:
this i think should be 14.5% instead
```suggestion
GTC_SELL_PCT = 0.145
```

Reply:
updated in fdcaa9d